### PR TITLE
feat(compute): the right Instance for learning/development/production purposes

### PR DESCRIPTION
**Pull request by RoRoJ**
### Description
Added three reference pages, one for each range of Instances

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: compute/instances/reference-content/cost-optimized.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  description: Find out more about Instances from Scaleway's Cost-Optimized range, such as Development and General Purpose Instances.
+content:
+  h1: The right Instance for development purposes
+  paragraph: Find out more about Instances from Scaleway's Cost-Optimized range, such as Development and General Purpose Instances.
+dates:
+  validation: 2023-02-22
+  posted: 2023-02-22
+categories: 
+  - compute
+tags: instance, type, development, DEV, range, cost-optimized, general-purpose, general, purpose, GP, PLAY2, PRO2, play, pro
```
Comment (nedaehm) [inline compute/instances/reference-content/cost-optimized.mdx:13]: I think we don't use commas in the tags section ?

File: compute/instances/reference-content/learning.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  description: Find out more about Instances from Scaleway's Learning range, such as the Stardust Instance.
+content:
+  h1: The right Instance for learning purposes
+  paragraph: Find out more about Instances from our Learning range, such as the Stardust Instance.
+dates:
+  validation: 2023-02-22
+  posted: 2023-02-22
+categories: 
+  - compute
+tags: instance, type, stardust, range, learning, use-case
```
Comment (nedaehm) [inline compute/instances/reference-content/learning.mdx:13]: same comment about commas in the tags section

File: compute/instances/reference-content/production-optimized.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  description: Find out more about Instances from Scaleway's Production-Optimized range, such as Enterprise Instances.
+content:
+  h1: The right Instance for production purposes
+  paragraph: Find out more about Instances from Scaleway's Production-Optimized range, such as Enterprise Instances.Purpose Instances.
+dates:
+  validation: 2023-02-22
+  posted: 2023-02-22
+categories: 
+  - compute
+tags: instance, type, production, production-optimized, range, ENT1, enterprise
```
Comment (nedaehm) [inline compute/instances/reference-content/production-optimized.mdx:13]: same comment about commas in the tags section

File: compute/instances/reference-content/cost-optimized.mdx (hunk 1 above)
Comment (ldecarvalho-doc) [reply compute/instances/reference-content/cost-optimized.mdx:13]: you're right je vais fix

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**